plugins/BEdita/API/tests/TestCase/Controller/ObjectsControllerTest.php

```diff
@@ -128,12 +129,14 @@ public function testIndex()
                             'links' => [
                                 'related' => 'http://api.example.com/documents/2/test',
                                 'self' => 'http://api.example.com/documents/2/relationships/test',
+                                'available' => 'http://api.example.com/objects?filter%5Btype%5D%5B0%5D=documents&filter%5Btype%5D%5B1%5D=profiles',
```

Comment:
can we use the more readable version `filter[type][]=documents&filter[type][]=profiles` ?

Reply:
@stefanorosanelli I should have fixed it using the solution suggested by Alberto.